exercises/bob/bob_test.f90

```diff
@@ -0,0 +1,71 @@
+
+! This test was created from ..\..\exercism\problem-specifications\exercises\bob\canonical-data.json
+! version: 1.4.0 
+!
+
+

```

Comment:
Too many new lines here in my opinion, as it should be one new line.  The same thing happens in other areas.

Reply:
I will add comments similar to in hello_world to guide new students